environments/servlet/tests/jetty/pom.xml

```diff
@@ -314,9 +314,9 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <excludes>
-                                <exclude>%regex[.*servlet/test/(?!se).*]</exclude>
-                            </excludes>
+                            <includes>
```

Comment:
So here's one interesting thing.  I was trying to figure out how to run the tests, these tests weren't working due to bad regex.  I don't think this is quite right, but let me know.  

One other thing - even with this change, I'm still seeing failures (but it works fine when I test with a test app).  For some reason, the underlying `WeldManager`'s ID is based on a path, not the UUID.